src/cascadia/TerminalSettingsEditor/SettingContainerStyle.xaml

```diff
@@ -0,0 +1,65 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under
+the MIT License. See LICENSE in the project root for license information. -->
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Microsoft.Terminal.Settings.Editor"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+    <Style x:Key="SettingContainerHeaderStyle" TargetType="StackPanel">
+        <Setter Property="Orientation" Value="Horizontal"/>
+        <Setter Property="Margin" Value="0,0,0,4"/>
+    </Style>
+
+    <Style x:Key="SettingContainerResetButtonStyle" TargetType="Button">
+        <Setter Property="Margin" Value="5,0,0,0"/>
+        <Setter Property="Height" Value="19"/>
+        <Setter Property="Width" Value="19"/>
+        <Setter Property="Padding" Value="0"/>
+        <Setter Property="Background" Value="Transparent"/>
+    </Style>
+
+    <Style x:Key="SettingContainerFontIconStyle" TargetType="FontIcon">
+        <Setter Property="Foreground" Value="{StaticResource SystemAccentColor}"/>
+        <Setter Property="FontSize" Value="14"/>
+        <Setter Property="FontFamily" Value="Segoe MDL2 Assets"/>
+    </Style>
+
+    <Style x:Key="SettingContainerOverrideTextStyle" TargetType="TextBlock">
+        <Setter Property="FontStyle" Value="Italic"/>
+    </Style>
+
+    <Style TargetType="local:SettingContainer">
+        <Setter Property="Margin" Value="0,24,0,0"/>
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="local:SettingContainer">
+                    <StackPanel>
+
+                        <!--Header space-->
+                        <StackPanel Style="{StaticResource SettingContainerHeaderStyle}">
+                            <TextBlock Text="{TemplateBinding Header}"/>
+                            <Button x:Name="ResetButton"
+                                    Style="{StaticResource SettingContainerResetButtonStyle}">

```

Comment:
oh I see, was it not possible to use an `x:Uid` here?

Reply:
(re: the comment about `HelpText`)

Reply:
Closing this thread in favor of the other one.